apollo-router-core/src/services/mod.rs

```diff
@@ -375,7 +375,7 @@ impl QueryPlannerResponse {
     ) -> Result<QueryPlannerResponse, BoxError> {
         tracing::warn!("no way to propagate error response from QueryPlanner");
         Ok(QueryPlannerResponse::new(
-            Arc::new(Default::default()),
+            Arc::new(QueryPlan::fake_new(None, None)),
```

Comment:
It's neater to use the builder.
```suggestion
            Arc::new(QueryPlan::fake_builder().build()),
```

Reply:
fixed in [a991a79](https://github.com/apollographql/router/pull/989/commits/a991a797c7f0b3da684dd6d3726cd18522161d8a)